src/app/components/Checkbox.tsx

```diff
@@ -32,7 +32,7 @@ const CustomCheckbox = styled(
   }
 
   input:checked + & {
-    background-color: ${theme.color.primary.orange.darkest};
+    background-color: ${(props) => props.disabled ? theme.color.primary.gray.base : theme.color.primary.orange.darkest};
```

Comment:
don't you want to set the disabled style regardless of whether its checked or not?

Reply:
no, because this sets the background of the checked box gray when disabled, which is only if it's in logged out state, that's what's in the design

Reply:
yea for the background, but i think the border needs to be changed when disabled as well